rel/app/env.bat.eex

```diff
@@ -1,4 +1,5 @@
 set RELEASE_MODE=interactive
+set LIVEBOOK_SHUTDOWN_ENABLED=true
```

Comment:
Maybe we should set it in the Dockerfile, this way it's just a single place?

Reply:
We want to enable only for the desktop app and dev, nowhere else. :)

Reply:
Ahh I was thinking local Docker, but we don't really know where it runs!